pkg/storage/couchdb/couchdbstore.go

```diff
@@ -67,6 +67,11 @@ func NewProvider(hostURL string, opts ...Option) (*Provider, error) {
 		return nil, fmt.Errorf(failToInstantiateKivikClientErrMsg, err)
 	}
 
+	ready, err := client.Ping(context.TODO())
```

Comment:
@sandrask , any idea if this should be `context.TODO()` or `context.Background()` (or something else??)

In the past I used context.Background and left a ToDo to make it customizable

Reply:
context.Background() makes more sense - I am not sure if you need to customize it

Reply:
fixed